config/settings/base.py

```diff
@@ -376,5 +376,25 @@
 }
 # SIMULATION
 # ------------------------------------------------------------------------------
+SIM_API_HOST = os.getenv(
+    "SIM_API_HOST", "https://optimizer-offgridplanner-app.apps2.rl-institut.de"
```

Comment:
I think we should rebase and remove this explicit URL, it is better to provide it via email